gulpfile.js

```diff
@@ -83,6 +93,13 @@ var webpackConfig = {
       }
     ]
   },
+  plugins: [
+    new webpack.NormalModuleReplacementPlugin(/\/config\/config(\.js)?$/,
+      function (result) {
+        result.request = configFilePath;
+      }
+    )
+  ],
```

Comment:
This addition of `NormalModuleReplacementPlugin` is needed due to the way imports work.